### DISCUSSION TITLE
How to contact support if you have lost your only 2fa recovery method?

### DISCUSSION THREAD

**Discussion by dhiller** [community/community#102618] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

A colleague's account was locked recently due to suspicious activity. He's had the 2fa recovery method by phone enabled, but he's getting message

```
We tried sending an SMS to your configured number, but we are not authorized to send SMS messages to this recipient. Please contact support if you continue to have problems.
```

now.

**Please note that he currently is logged out and unable to login into his account, so https://support.github.com/ will not work for him!**

Any clue how else he could contact support to resolve this?

**Comment by NittuzDev** [comment]
Try to go here: https://support.github.com/
Then go in down-left corner and click on Contact Us. Then click on 2FA issues

**Reply by dhiller** [reply]
IIUC that option only works when you are logged in, which is not the case for him.

**Comment by ADKATT** [comment]
Hello @dhiller 

If you've lost access to your two-factor authentication (2FA) recovery methods on GitHub and are unable to sign in, GitHub provides a specific account recovery process to help you regain access. 

Follow these steps to contact GitHub support:

Go to GitHub Support:
Open the GitHub Support page in your web browser:
https://support.github.com/request

Choose the Relevant Category:
In the support form, you'll be asked to select a category for your issue. Choose the option related to "Two-factor authentication" or "Account recovery."

Provide Necessary Information:
Fill out the form with as much information as possible.
Include details about your account, the issue you're facing, and any relevant information that can help GitHub verify your identity.

Submit the Request:
After filling out the form, submit your request.

Wait for GitHub's Response:
GitHub support will review your request, and they may ask for additional information to verify your identity.
Be patient and responsive to their requests.

Thanks

**Reply by dhiller** [reply]
I've used the above page with an anonymous browser session, it seems you can then choose "I can't login" and are transferred to the page: https://support.github.com/contact/cannot_sign_in

There you can choose "Account locked".

I've forwarded that to my colleague, keeping :crossed_fingers:
